app/src/main/java/com/sun/weather/ui/home/HomeFragment.kt

```diff
@@ -50,6 +47,9 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(FragmentHomeBinding::infl
                 }
             }
         }
+        binding.constraintLayout.setOnClickListener {
+            replaceFragment(R.id.fragment_container, DetailFragment.newInstance(cityName!!), true)
```

Comment:
```suggestion
            replaceFragment(R.id.fragment_container, DetailFragment.newInstance(cityName!!), true)
```
Không force unwrap !!